### PR TITLE
Renamed Vault functions and events with Ousd in them

### PR DESCRIPTION
**Pull request by naddison36**
This is an optional addition to the PR https://github.com/OriginProtocol/origin-dollar/pull/1627 that refactors the vaults

It does more more of the items in https://github.com/OriginProtocol/origin-dollar/issues/1533
Specifically, it covers removing `Ousd` from function names and events.
- `setOusdMetaStrategy` -> `setMetaStrategy`
- `setNetOusdMintForStrategyThreshold` -> `setNetMintForStrategyThreshold`
- `OusdMetaStrategyUpdated` -> `MetaStrategyUpdated`
- `NetOusdMintForStrategyThresholdChanged` -> `NetMintForStrategyThresholdChanged`

The two functions and corresponding events are only used by the governor so breaking the ABI is not such a big deal

### CODE REVIEW

File: contracts/contracts/interfaces/IVault.sol [hunk 1]
```diff
@@ -165,9 +165,9 @@ interface IVault {
 
     function netOusdMintForStrategyThreshold() external view returns (uint256);
 
-    function setOusdMetaStrategy(address _ousdMetaStrategy) external;
+    function setMetaStrategy(address _strategy) external;
```
Comment (shahthepro) [inline contracts/contracts/interfaces/IVault.sol:168]: We don't plan to use this for OETH though right? This is exclusive to OUSDMetStrategy contract. the curve AMO uses mintForStrategy I believe

Comment (naddison36) [reply contracts/contracts/interfaces/IVault.sol:168]: It currently is on the OETHVaultAdmin hence the reason for are trying to clean it up.

There may be a better way of doing this. eg
1. Move these OUSD specific functions to an OUSD Vault implementation contract
2. Change the OUSD strategies to use the more generic functions.

Both sound like a bigger change for another day.

**Comment by sparrowDom** [comment]
@naddison36 do we want to pair this PR with deployment of some other Vault functionality? So we don't kick off the Vault deploys just for the naming changes?

**Comment by sparrowDom** [comment]
Maybe it'b be cool to have a list of PRs that are waiting to join some other more impactful functionality to be deployed

**Comment by naddison36** [comment]
For the OETH Vault, I'd say it's best to pair with something else.
For the OUSD Vault, we need to decide if we include with `052_upgrade_ousd_vault_harvester` or do after. I'm not set on either way but I'd lean toward doing after so both OETH and OUSD are on the same codebase.

**Comment by sparrowDom** [comment]
Yes agreed, rolling them out for both protocols simultaneously would be nice - in respect of not complicating our environments and mental load.

**Comment by naddison36** [comment]
I think OUSD has been baked in now

**Pull request closed without merging**